day01_assignment/2.Variables/question1.js

```diff
@@ -0,0 +1,15 @@
+// Question 1
+
+// Declare a variable - let a;. On another line assign any value you like to a. 
+// Log the value of a before and after assignment. Try doing the same with const. 
+
+
+let a
+console.log("For let:");
+console.log(a);
+a = 10
+console.log(a);
+
+const b = 12
+console.log("For const:");
+console.log(b);
```

Comment:
give outputs in the end of each assignment/question in comments so it'll be easy to understand your understanding of concept and review the code too